### DISCUSSION TITLE
zh/guide/drivers/teambition

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#125] [Announcements]
# zh/guide/drivers/teambition

WebDAV 配置 Name Value -------- --------------------------- Url http[s]\://domain:port/dav/ Host domain 路径 dav 协议 http/https 端口 与网页端一致 WebDAV用户名 与网页端用户名一致 WebDAV密码 与网页端密码一致 | WebD...

https://alist.nn.ci/zh/guide/drivers/teambition.html

**Comment by Elense420** [comment]
复制文件需要S-1-1-0权限是什么？怎么解决😭

**Comment by simplove** [comment]


**Reply by 65887166** [reply]
问题这个是teambition，不是Trainbit

**Comment by simplove** [comment]
> 问题这个是teambition，不是Trainbit
确实，大意了

**Comment by Aimer52000** [comment]
有人知道 teambition 项目有容量限制不

**Comment by shijianyu00** [comment]
下载已经收到了限制，需要升级版本，但是点击去之后可以下载。请修复一下感谢

**Reply by goldsoft8888** [reply]
可以上传，不能下载，请修复

**Reply by goldsoft8888** [reply]
官方回复：我这边核实了下，免费版本目前限制了传输量，免费企业每天所有人员上传上限是 20G/天，一个月是 200G 上限 ，下载也有限制，免费企业当日是20G ，单用户是5G
---------------------------------------
实际测试中，5G也达不到

**Reply by goldsoft8888** [reply]
实际测试，应该是5G流量，折合就是下载一个500兆文件，一天流量就用完了

**Reply by yohototo** [reply]
不限的，只限制单文件5g
